packages/cli/src/commands/deviceprofiles/create.ts

```diff
@@ -120,7 +120,6 @@ export async function createWithDefaultConfig(client: SmartThingsClient, data: D
 export function cleanupRequest(deviceProfileRequest: Partial<DeviceProfile & { restrictions: unknown }>): DeviceProfileRequest {
 	delete deviceProfileRequest.id
 	delete deviceProfileRequest.status
-	delete deviceProfileRequest.owner
```

Comment:
What was the driver behind removing this?

Reply:
I removed the field from the model object in https://github.com/SmartThingsCommunity/smartthings-core-sdk/pull/101 because it was included erroneously. (It's not in the docs and doesn't get included in the payload.)